### PR TITLE
[Ready] Re-balance Tech Web nodes and points

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Adds in Adv Lugs (There missing or im blind)
Lots of point balances and a few nodes getting lesser/more tech nodes required before you can get them 
*side note this will help antags get tech nodes faster, in case one has to steal like 30 nodes in 30 mins...*
[why]
Politely speeds up progression with parts and basic weapons well still making sure that other nodes that are intended to be late/early game stay around that kinda of area.

### CODE REVIEW

**Comment by deathride58** [comment]
A lot of these feel arbitrary, as a lot of these value adjustments result in an overall total point requirement decrease for the majority of practical nodes. The only nodes this PR increases the total point requirements of are the weapon nodes. The fix for the inconsistency where bluespace bodybags are available in a node that isn't bluespace holding tech on top of actually being present in bluespace holding tech can and should probably be atomized.

**Comment by Trilbyspaceclone** [comment]
_A lot of these feel arbitrary, as a lot of these value adjustments result in an overall total point requirement decrease for the majority of practical nodes._
That's kinda the point, to speed up the branching paths. You cant get much tech form 50k bombs. The whole point is to politely speed up things. Most gear and parts that are late game are still hour way and mid parts are around 30~
_The_only nodes this PR increases the total point requirements of are the weapon nodes._
Thats to stop people form getting late game items before the 1 hour mark. Like X-ray, Ele and mark gun as are all powerful guns. So they should be late game. But the Adv wep techs are also branches into protects and other borg tech.
_The fix for the inconsistency where bluespace bodybags are available in a node that isn't bluespace holding tech_
If I was to power game I would not fix that. Those are some of the most bullshit weapons in the game.
_on top of actually being present in bluespace holding tech can and should probably be atomized._
I mean, Other then Boh in that no one really minds it form what I hear/seen. Could be wrong, if I am then ill remove it and resemblance the BSBB under a different late game node

**Review by deathride58 — CHANGES_REQUESTED** [review]
Advanced Data Theory should be increased from 2500 points to 3000 points to accommodate for Data Theory's cost decrease from 2500 to 2000.
Industrial Engineering's cost can be rounded up to 6000, and Advanced Engineering can be increased to 4000 points.
High Efficiency Parts can be rounded down to 5000, and Advanced Bluespace Research should be increased to 7500 to make up for the overall point decrease
Advanced Plasma Research should have its cost increased to 3000 to make up for Basic Plasma Research's cost decrease to 2000
Cyborg Upgrades: Utility's cost decrease to 1750 should be rolled back to its original value of 2000 due to the usefulness of the designs unlocked by it

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -192,7 +192,7 @@
 	prereq_ids = list("bluespace_basic", "engineering")
 	design_ids = list("bs_rped","minerbag_holding", "bluespacebeaker", "bluespacesyringe", "phasic_scanning", "roastingstick", "ore_silo")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
-	export_price = 5000
+	export_price = 7500
```
Comment (deathride58) [inline code/modules/research/techweb/all_nodes.dm]: ?????
this isnt even the right var

Comment (Trilbyspaceclone) [reply code/modules/research/techweb/all_nodes.dm]: huh

**Review by deathride58 — APPROVED** [review]

**Pull request merged**